Description: bionic pod2man fails on empty release options
 Ubuntu 18.04 bionic fails to produce valid man pages when pod2man is run with
 --release="". As a fix we remove the option completely.
 it.
 .
 cobbler (2.8.4-1) unstable; urgency=low
 .
   * Maintenance release in the Cobbler 2.8 series
Author:  Jorgen Maas <jorgen.maas@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2019-04-21

--- cobbler-2.8.4.orig/setup.py
+++ cobbler-2.8.4/setup.py
@@ -305,7 +305,7 @@ class build_man(Command):
                 # It is. Configure it
                 self.build_one_file(infile, outfile)
 
-    _COMMAND = 'pod2man --center="%s" --release="" %s | gzip -c > %s'
+    _COMMAND = 'pod2man --center="%s" %s | gzip -c > %s'
 
     def build_one_file(self, infile, outfile):
         man = os.path.splitext(os.path.splitext(os.path.basename(infile))[0])[0]
